Emagging cargo console also unlocks contraband

coiax opened the pull request:
Using an emag will also unlock some of the speciality crates that you would
normally have to multitool the cargo circuit board for.

:cl: coiax
rscadd: Emagging the cargo console also unlocks regular contraband, in addition to syndicate gear
/:cl:

coiax committed:
Emagging cargo console also unlocks contraband

Using an emag will also unlock some of the speciality crates that you would
normally have to multitool the cargo circuit board for.

Supermichael777 commented:
i always thought it did this.

Supermichael777 commented on code/modules/cargo/console.dm:
```diff
@@ -24,6 +24,11 @@
 
+		// Emagging also unlocks the Broad-banded supplies
+		var/obj/item/weapon/circuitboard/cargo/board = circuit
+		circuit.contraband = TRUE
```
needs to be board.contraband
also the thing should become permanently emagged and broad-banded

coiax commented:
Permanently emagging and broadbanding the board sounds like a good idea. Emagging is stored on the console though, I'll have to switch some things around.

coiax committed:
Added emagging the cargo circuit board

In addition, a circuit once emagged will be unable to return to
only legitimate channels.

coiax committed:
You can now only emag the cargo circuit once
